src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String CIRCLE_COLOR = "WHITE";
+    private static final int CIRCLE_RADIUS = 10;
+    private static final Random random = new Random();
```

Comment:
The `random` object should not be static because it's not a constant and it's related to the state of the `FigureSupplier` instance.